src/components/SendTransaction/SetFee.vue

```diff
@@ -95,16 +91,17 @@ export default {
   },
   data() {
     const formValidation = () =>
-      new FormValidation({ unit: this.unit, balance: this.availableBalance })
+      new FormValidation({
+        unit: this.unit,
+        balance: this.availableBalance,
+        feeType: this.feeType,
+      })
 
     const maxNumber = (rule, value, callback) => {
       return formValidation().maxNumber(rule, value, callback)
     }
-    const integerNanoWit = (rule, value, callback) => {
-      return formValidation().integerNanoWit(rule, value, callback)
-    }
-    const minAmount = (rule, value, callback) => {
```

Comment:
Why are we removing the min amount validation?

Reply:
yes, just in the custom miner fee input. It allows sending 0 nanowit.